test/lib.similar.js

```diff
@@ -8,4 +8,8 @@ describe('Similar method', () => {
     return store.similar({id: '553834731'})
     .then((apps) => apps.map(assertValidApp));
   });
+  it('should fetch a valid application list in fr country', () => {
```

Comment:
Well, you could make a test that the results are actually different between default and fr country, maybe? Otherwise there's no way to know this works, if it's not taking the country from something machine specific or that future commits don't break it.